src/remote/index.js

```diff
@@ -119,11 +119,16 @@ export default class Remote implements Side {
   async overwriteFileAsync (doc: Metadata, old: ?Metadata): Promise<Metadata> {
     log.info(`${doc.path}: Uploading new file version...`)
     const stream = await this.other.createReadStreamAsync(doc)
-    const updated = await this.remoteCozy.updateFileById(doc.remote._id, stream, {
+    const options = {
       contentType: doc.mime,
       checksum: doc.md5sum,
-      lastModifiedDate: new Date(doc.updated_at)
-    })
+      lastModifiedDate: new Date(doc.updated_at),
+      ifMatch: ''
```

Comment:
Oh yeah... Flow considers object litterals as "closed" by default.
If you declare with `const options: Object = ...`, I believe it should work (although I agree specifying that `options` is an `Object` is not really valuable...).